tests/installation/partitioning_filesystem.pm

```diff
@@ -25,6 +25,8 @@ sub run {
     assert_screen 'edit-proposal-settings';
     wait_screen_change { send_key $cmd{guidedsetup} };
 
+    select_first_hard_disk if (check_screen 'select-hard-disks', 0);
```

Comment:
I'm not sure if that's exactly what you need as we will end up with configured disk. Also see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4912 There I've extracted whole method there...